INFO     2025-08-07 20:33:50 [lingkuan_801UAT.conftest;197]  : [2025-08-07 20:33] 测试会话开始: 2025-08-07 20:33:50
INFO     2025-08-07 20:33:50 [lingkuan_801UAT.commons.variable_manager;39]  : [2025-08-07 20:33] 成功加载静态变量: VAR\test_data.json
INFO     2025-08-07 20:33:50 [lingkuan_801UAT.commons.variable_manager;55]  : [2025-08-07 20:33] 成功加载运行时变量: VAR\runtime_vars.json
INFO     2025-08-07 20:33:50 [lingkuan_801UAT.conftest;64]  : [2025-08-07 20:33] 用例 test_vps/test_create.py::TestCreate::test_create_user 使用默认URL登录: https://uat.atcp.top/api
INFO     2025-08-07 20:33:50 [lingkuan_801UAT.conftest;74]  : 登录尝试 1/3，生成MFA验证码: 361047
WARNING  2025-08-07 20:33:50 [lingkuan_801UAT.conftest;108]  : 第1次登录失败: HTTPSConnectionPool(host='uat.atcp.top', port=443): Max retries exceeded with url: /api/sys/auth/login (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))
INFO     2025-08-07 20:33:50 [lingkuan_801UAT.conftest;110]  : 等待 15 秒后重试...
INFO     2025-08-07 20:33:54 [lingkuan_801UAT.conftest;224]  : [2025-08-07 20:33] 测试会话结束，总耗时: 4.00秒
INFO     2025-08-07 20:33:54 [lingkuan_801UAT.commons.feishu_notification;128]  : [2025-08-07 20:33] 发送飞书消息，URL: https://open.feishu.cn/open-apis/bot/v2/hook/8d3475ac-8adc-45ed-97c7-0f0ec8647a4f
